.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Needs: planning, Type: bug'
+assignees: benjamincharity
+
+---
+
+#### 1. What is the expected behavior?
+
+
+#### 2. What is the current behavior?
+
+
+#### 3. What are the steps to reproduce?
```

Comment:
This is my bad... I wasn't thinking about the actual steps in the file.. I think for this project we can remove step 3 and step 4. (since this is a deployed site, there's not much the user can do that we need to replicate etc)